docs/user-docs/install_aem.md

```diff
@@ -93,18 +107,45 @@ sudo qubes-dom0-update --disablerepo="*" --enablerepo=aem --action=reinstall \
     grub2-tools-minimal
 ```
 
-### Updating GRUB
+###### Updating GRUB on legacy systems
 
-Booting on legacy systems (AEM currently doesn’t support UEFI) requires manual
-installation of GRUB2 to the MBR of disk where Qubes OS is stored. In the
-example below it is `/dev/sda`, yours may be different. Remember that GRUB2
-must be installed on disk and not on partition, so don’t use `sda1`, `nvme0n1p1`
-etc.
+Booting on legacy systems requires manual installation of GRUB2 to the MBR
+of disk where Qubes OS is stored. In the example below it is `/dev/sda`,
+yours may be different.
+
+To check on which partition is your OS installed run:
+
+```bash
+df --output=source /
+```
+
+Remember that GRUB2 must be installed on disk and
+not on partition, so don’t use `sda1`, `nvme0n1p1` etc. This step should be
+skipped on UEFI systems.
 
 ```bash
 sudo grub2-install /dev/sda
 ```
 
+##### UEFI Systems
+
+If your system has an UEFI BIOS, install these packages instead:
+
+```bash
+sudo qubes-dom0-update --disablerepo="*" --enablerepo=aem --action=reinstall \
+    python3-xen \
+    xen \
+    xen-hypervisor \
+    xen-libs \
+    xen-licenses \
+    xen-runtime \
+    grub2-common \
+    grub2-efi-x64 \
+    grub2-efi-x64-modules \
```

Comment:
I'm getting `No match for argument` for this package, it should be installed, not reinstalled.